### PR TITLE
Integrate w3up for `ipfs-pinner`

### PR DESCRIPTION
**Pull request by noslav**

### CODE REVIEW

File: config/dev.exs [hunk 1]
```diff
@@ -50,7 +50,7 @@ config :phoenix, :plug_init_mode, :runtime
 # Default these should point to moonbase alpha since we do all the development testing there
 config :rudder,
   operator_private_key: System.get_env("BLOCK_RESULT_OPERATOR_PRIVATE_KEY"),
-  bsp_proofchain_address: "0x1BFa3b5E9bE2c5298B7DE11B5Acb08c37683f4eF",
+  bsp_proofchain_address: "0x30F220B44E937dd6A2A43D91D564E259f3574eb8",
   brp_proofchain_address: "0x06A38f70c8e245e1cfe1b7d5ACa1B7cd29380343",
```
Comment (rogarcia) [inline config/dev.exs:54]: for moonbase alpha the brp proofchain was also upgraded to the v1.5 contract before the break. See the PR I pushed
https://github.com/covalenthq/rudder/pull/174/files

https://www.notion.so/covalenthq/sepolia-deployment-of-contract-816b451b9e754ae2b4e81c532943d2f1?pvs=4#07611302e52c479fa5bbc93c6eba5593

Comment (noslav) [reply config/dev.exs:54]: the reason I didn't update this was because of the operators and validators weren't added on the new contract

Comment (noslav) [reply config/dev.exs:54]: we also need to update the event names as the contracts were upgraded

Comment (noslav) [reply config/dev.exs:54]: lets do that on a new feature branch dedicated to testing on moonbase

File: docker-compose-ci.yml [hunk 2]
```diff
@@ -16,6 +16,7 @@ services:
       - cqt-net
     environment:
       - ERIGON_NODE=${ERIGON_NODE}
+      - NODE_TLS_REJECT_UNAUTHORIZED=0
```
Comment (noslav) [inline docker-compose-ci.yml:19]: 🥇

**Pull request merged**